diff --git a/node_modules/@types/oidc-provider/index.d.ts b/node_modules/@types/oidc-provider/index.d.ts
index 8462609..a7b1c24 100755
--- a/node_modules/@types/oidc-provider/index.d.ts
+++ b/node_modules/@types/oidc-provider/index.d.ts
@@ -984,12 +984,12 @@ export interface Configuration {
             } | undefined;
             idFactory?: ((ctx: KoaContextWithOIDC) => string) | undefined;
             secretFactory?: ((ctx: KoaContextWithOIDC) => string) | undefined;
+            issueRegistrationAccessToken?: IssueRegistrationAccessTokenFunction | boolean | undefined;
         } | undefined;
 
         registrationManagement?: {
             enabled?: boolean | undefined;
             rotateRegistrationAccessToken?: RotateRegistrationAccessTokenFunction | boolean | undefined;
-            issueRegistrationAccessToken?: IssueRegistrationAccessTokenFunction | boolean | undefined;
         } | undefined;
 
         deviceFlow?: {
