diff --git a/node_modules/@types/oidc-provider/index.d.ts b/node_modules/@types/oidc-provider/index.d.ts
index 858b7b1..1d3820e 100755
--- a/node_modules/@types/oidc-provider/index.d.ts
+++ b/node_modules/@types/oidc-provider/index.d.ts
@@ -823,6 +823,9 @@ export interface AdapterPayload extends AllClientMetadata {
     lastSubmission?: InteractionResults | undefined;
     loginTs?: number | undefined;
     nonce?: string | undefined;
+    openid?: {
+      scope?: string,
+    },
     params?: UnknownObject | undefined;
     policies?: string[] | undefined;
     redirectUri?: string | undefined;
@@ -984,12 +987,12 @@ export interface Configuration {
             } | undefined;
             idFactory?: ((ctx: KoaContextWithOIDC) => string) | undefined;
             secretFactory?: ((ctx: KoaContextWithOIDC) => string) | undefined;
+            issueRegistrationAccessToken?: IssueRegistrationAccessTokenFunction | boolean | undefined;
         } | undefined;
 
         registrationManagement?: {
             enabled?: boolean | undefined;
             rotateRegistrationAccessToken?: RotateRegistrationAccessTokenFunction | boolean | undefined;
-            issueRegistrationAccessToken?: IssueRegistrationAccessTokenFunction | boolean | undefined;
         } | undefined;
 
         deviceFlow?: {
